server/static/main.html

```diff
@@ -5,51 +5,86 @@
     <meta http-equiv="content-type" content="text/html; charset=UTF-8">
     <meta charset="UTF-8">
     <title>Unshort.link- Free link unshort service</title>
+    <link href="/static/css/bootstrap.min.css" rel="stylesheet" type="text/css">
     <link href="/static/css/reset.css" rel="stylesheet" type="text/css">
     <link href="/static/css/style.css" rel="stylesheet" type="text/css">
+    <script src="https://kit.fontawesome.com/cb03664470.js" crossorigin="anonymous"></script>
```

Comment:
Please also no external font awesome. If there is a local alternative we can include into the Repo (should be AGPL compatible) that would be a lot better

Reply:
Material icons could be an option: https://material.io/resources/icons/?style=baseline

Reply:
Annoyingly enough, only Font Awesome has the Firefox icon, so I'd have to omit the extension icons. 

Currently, I'm trying [Feather Icons](https://feathericons.com/) because it uses the MIT license.

Reply:
Sorry for not checking it earlier: but apparently font awesome is GPL compatible, so we can use it https://github.com/FortAwesome/Font-Awesome/blob/master/LICENSE.txt

Reply:
It somewhat comes down to your preference. Whether the former or latter version is your choice. Though, using Font Awesome would make the most sense, as it allows for the extensions to be listed at the bottom.

Cheers,
Natamo.

Reply:
Would be nice to have the extensions listed.

Reply:
I'll fix that up and commit it.

Cheers,
Natamo